roles/download/defaults/main/checksums.yml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
@VannTen Hello, may I ask if this file is redundant? The actual checksum information is stored in the `roles/download/defaults/main.yml` file. I'm not quite sure what the purpose of this new file is.

Reply:
Hum, yeah, not sure how this got there. :thinking:

Reply:
😊Do you have time to verify if this file is useless? If it is, I suggest removing it. Thank you very much.🤝

Reply:
Hum it does not appear to be in release-2.22 though.. :thinking: